automated-testing/readme.md

```diff
@@ -0,0 +1,16 @@
+# Testing
+
+## Sections within Testing
+
+* [Unit testing](unit-testing/readme.md)
+* [Unit testing v2](unit-testing-v2/readme.md)
```

Comment:
What would be the difference between unit testing v2 and unit testing? Seems strange to me to have two sections talking about the same area differently.

Reply:
Hello @fnocera ,

We did not want to modify the existing Unit Testing version before the new one is created. That is why we created new page for the new version we will be working on. 

Thank you!

Reply:
I agree with @fnocera here, leaving the docs in an "in-between" state is a bit risky as many people are using these docs.